internal/generator/gen_test.go

```diff
@@ -4,19 +4,20 @@ import (
 	"io"
 	"log"
 	"os"
-	"path"
 	"path/filepath"
 	"strings"
 	"testing"
 
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/woven-planet/go-zserio/internal/ast"
 	"github.com/woven-planet/go-zserio/internal/model"
 )
 
 func testWorkspace(filePath string) string {
-	return path.Join(os.Getenv("TEST_SRCDIR"), os.Getenv("TEST_WORKSPACE"), filePath)
+	actualPath, _ := bazel.Runfile(filePath)
```

Comment:
I think we need to catch errors returned by `Runfile()`, and perhaps panic if those happen. I'm hoping that will help us detect why the tests are failing.

```suggestion
	actualPath, err := bazel.Runfile(filePath)
	if err != nil {
	    panic(err)
	}
```

Reply:
I think the best way would be to use `require.NoError(t, err)` instead of panicking. That would require passing `t *testing.T` to the function, but that may be quite a good thing, I think.